tutor/commands/config.py

```diff
@@ -220,8 +224,32 @@ def patches_list(context: Context) -> None:
     renderer.print_patches_locations()
 
 
+@click.command(name="edit", help="Edit config.yml of the current environment")
+@click.pass_obj
+def edit(context: Context) -> None:
+    config_file = os.path.join(context.root, "config.yml")
+
+    open_cmd = None
+
+    if which("open"):  # MacOS
+        open_cmd = ["open", config_file]
+    elif which("xdg-open"):  # Linux
+        open_cmd = ["xdg-open", config_file]
+    elif which("start"):  # Windows
+        open_cmd = ["start", '""', config_file]
+    else:
+        click.echo(
+            "Cannot find a way to open the editor automatically. Kindly open the file manually: "
+            + config_file
+        )
+        return
+
+    subprocess.call(open_cmd)
```

Comment:
This command will do something funky if the file or its directory does not exist. I suggest the command should fail if the config file does not exist.

Reply:
Great catch! The `open`s I tested all launched the browser with the argument as the URL.